README.md

```diff
@@ -14,7 +14,7 @@ Currently available only for scala 2.11. In your build.sbt file,
 ```sbt
 resolvers += Resolver.jcenterRepo
 
-libraryDependencies += "io.ticofab" % "aws-request-signer_2.11" % "0.4.1"
+libraryDependencies += "io.ticofab" %% "aws-request-signer" % "0.4.1"
```

Comment:
if we do this, and someone uses this library from a scala 2.12 project, the library won't be found.